app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
```

Comment:
what is the difference between a User and a Player?

Reply:
That's a good question. That configuration was set up by Jose, so I essentially went along with it. I would guess that a user is someone who merely logs in, while a "player" is someone who joins a game.

Reply:
Perhaps we can get rid of the Player table?

Reply:
@josereynaud What do you think of that suggestion? Just keep the Player model?

Reply:
Player was someone who logs in. I thought when @RMukerjee416 did the username he would add it to the Player table but he created a whole new table. In the end, if it's easier to just delete the Player model and just use the User model is fine by me.

Reply:
Thanks for the feedback. I have deleted the player model and did some rollbacks and migrations so that we are just working with the user model. I have pushed the changes to this branch for further review.